test/parallel/test-wrap-js-stream-exceptions.js

```diff
@@ -10,7 +10,7 @@ process.once('uncaughtException', common.mustCall((err) => {
 }));
 
 const socket = new JSStreamWrap(new Duplex({
-  read: common.mustNotCall(),
+  read: common.mustCall(),
```

Comment:
I don't quite understand this test or the JSWrap stuff... Throwing inside of a write seems rather wrong?

I hope this change is not of any significance.

Reply:
No, this change is okay.